RELEASE_NOTES

```diff
@@ -3,9 +3,42 @@ For more detailed information, please see the git log.
 
 These release notes can also be consulted at http://easybuild.readthedocs.org/en/latest/Release_notes.html.
 
+v2.8.2 (July 12th 2016)
+-----------------------
+
+bugfix release
+- various small enhancements, including:
+    - add support for rst output for --list-* and --avail-* (#1339)
+    - add support for 'eb --check-conflicts' (#1747, #1807, #1833)
+    - ensure nice error message when non-existing path is passed to apply_regex_substitutions (#1788)
```

Comment:
ok, got it, here's the URL asked:
http://easybuild.readthedocs.io/en/latest/Extended_dry_run.html#extended-dry-run-guidelines-easyblocks-framework-functions-runtime-patching